deis-monitor/manifests/deis-monitor-grafana-rc.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: deis-monitor-grafana
+  namespace: deis
+  labels:
+    heritage: deis
+spec:
+  replicas: 1
+  selector:
+    app: deis-monitor-grafana
+  template:
+    metadata:
+      labels:
+        app: deis-monitor-grafana
+    spec:
+      containers:
+      - name: deis-monitor-grafana
+        image: quay.io/deis/grafana:canary
+        imagePullPolicy: Always
+        env:
+        - name: "INFLUXDB_URLS"
+          value: http://$(DEIS_MONITOR_INFLUXAPI_SERVICE_HOST):$(DEIS_MONITOR_INFLUXAPI_SERVICE_PORT_TRANSPORT)
+        - name: "KUBERNETES_DOCKER_ENDPOINT"
+          value: "unix:///var/run/docker.sock"
```

Comment:
would it make sense to parameterize this so clusters who don't use this socket can configure this chart as necessary?